app/dashboard/templates/dashboard/hackathon/hackathons.html

```diff
@@ -25,14 +25,32 @@
     <style>
 
       img.card-top {
-        max-height: 6rem;
+        max-height: 8rem;
+        max-width: 11rem;
       }
 
       @media (min-width: 1200px) {
         .container.custom-container {
           max-width: 1400px;
         }
       }
+
+
+      @media (max-width: 920px) {
+        img.card-top {
+          max-width: 100%;
+        }
```

Comment:
same here, the problem is when we jump into a bundle this code will be loaded over the site and overwrite others components.